commands/auth_utils.go

```diff
@@ -29,25 +37,41 @@ type Credentials struct {
 
 type CredentialsList map[string]*Credentials
 
-func getConfigFilePath() (string, error) {
-	user, err := user.Current()
+func getDefaultConfigPath() string {
+	return filepath.Join("$HOME", xdgConfigHome, "mmctl")
```

Comment:
The reason I did this was to provide a proper help text in `root.go`, where the flags are defined. Otherwise, all docs generated on my machine, for instance, would contain the default value of `/home/ferdinan/.config` (i.e. my local username) for this argument.